packages/core/src/components/chip/_chip.scss

```diff
@@ -0,0 +1,70 @@
+@import '../../global/variables';
+
+$ray-chip-border-width: 1px;
+$ray-chip-height: $ray-spacing-xs * 2 + 1rem;
+
+.#{$ray-class-prefix}chip {
+  @include no-select;
+  border: $ray-chip-border-width solid $ray-color-blue-50;
+  background-color: $ray-color-white;
+  padding: $ray-spacing-xs $ray-spacing-md;
+  border-radius: $ray-chip-height;
+  color: $ray-color-blue-50;
+  display: inline-block;
+  cursor: pointer;
+
+  &:not(:last-child) {
+    margin-right: $ray-spacing-xs;
+
+    [dir='rtl'] & {
+      margin-right: 0;
+      margin-left: $ray-spacing-xs;
+    }
+  }
+
+  @media (hover: hover) {
+    &:hover {
+      background-color: $ray-color-blue-10;
+      color: $ray-color-blue-50;
+    }
+  }
+
+  @include ray-breakpoint(desktop) {
+    &:hover {
+      background-color: $ray-color-blue-10;
+      color: $ray-color-blue-50;
+    }
+  }
+
+  &:focus {
```

Comment:
Please put this behind the accessibility class (see https://github.com/wework/ray/blob/master/packages/core/src/components/button/_button.scss#L77 for an example).

We do this so the focus states dont appear on tap or click, which can be confusing for some people as they wont go away until the user focuses on another element